test/test.go

```diff
@@ -0,0 +1,7 @@
+package test
```

Comment:
plz remove this file

Reply:
My Mac won't allow me to right click and delete the file

Reply:
you can't delete files on your computer?